documentation/release_notes.rst

```diff
@@ -38,6 +38,8 @@ New in This Release
 ^^^^^^^^^^^^^^^^^^^
 - Incorrect results may be produced with in-place scans using ``unseq`` and ``par_unseq`` policies on
   CPUs with the Intel® C++ Compiler 2021.8.
+- Incorrect results may be produced with group reduce operation through ``std::reduce``
+  for 64-bit data types and `std::multiplies`` operation on GPUs with the Intel® C++ Compiler 2023.2
```

Comment:
I will check clarify the compiler versions

Reply:
In the file https://github.com/oneapi-src/oneDPL/blob/72551f4314b86f68df85661f8946d1309d2bc0ca/include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h#L44
we have the next code:
```C++
template <typename _BinaryOp, typename _Tp>
using __has_known_identity =
#    if _ONEDPL_LIBSYCL_VERSION >= 50200
    typename ::std::disjunction<
        __dpl_sycl::__has_known_identity<_BinaryOp, _Tp>,
        ::std::conjunction<::std::is_arithmetic<_Tp>,
                           ::std::disjunction<::std::is_same<::std::decay_t<_BinaryOp>, ::std::plus<_Tp>>,
                                              ::std::is_same<::std::decay_t<_BinaryOp>, ::std::plus<void>>,
                                              ::std::is_same<::std::decay_t<_BinaryOp>, __dpl_sycl::__plus<_Tp>>,
                                              ::std::is_same<::std::decay_t<_BinaryOp>, __dpl_sycl::__plus<void>>,
                                              ::std::is_same<::std::decay_t<_BinaryOp>, __dpl_sycl::__minimum<_Tp>>,
                                              ::std::is_same<::std::decay_t<_BinaryOp>, __dpl_sycl::__minimum<void>>,
                                              ::std::is_same<::std::decay_t<_BinaryOp>, __dpl_sycl::__maximum<_Tp>>,
                                              ::std::is_same<::std::decay_t<_BinaryOp>, __dpl_sycl::__maximum<void>>>>>;
```
Describing problem is in the function `__dpl_sycl::__has_known_identity<_BinaryOp, _Tp>,` whi called by in case when ` if _ONEDPL_LIBSYCL_VERSION >= 50200`
Unfortunately I not fund which version of DPC++ compiler correspondent with this `50200` constant...

Reply:
@SergeyKopienko, this macro version corresponds to Intel(R) oneAPI DPC++/C++ Compiler 2021.3.

Reply:
OK, let's specify as 2021.3